x-pack/legacy/plugins/reporting/export_types/common/execute_job/decrypt_job_headers.test.ts

```diff
@@ -7,19 +7,28 @@
 // @ts-ignore
 import { cryptoFactory } from '../../../server/lib/crypto';
 import { createMockServer } from '../../../test_helpers/create_mock_server';
-import { decryptJobHeaders } from './index';
+import { Logger } from '../../../types';
+import { decryptJobHeaders } from './decrypt_job_headers';
 
-let mockServer: any;
-beforeEach(() => {
```

Comment:
It looked weird to me that this isn't in a `describe`

Reply:
Yeah, this is an anti-pattern from what I recall.